pytorch_to_returnn/converter/converter.py

```diff
@@ -156,6 +156,14 @@ def _make_tf_feed_dict(self, input: Data):
     else:
       for i, size in input.size_placeholder.items():
         d[size] = [self._inputs_np.shape[input.get_batch_axis(i)]] * n_batch  # not so relevant
+    if out_entry:
+      def _output_from_non_deterministic(tensor_entry_: TensorEntry):
```

Comment:
I don't think it makes it really easier to read to make this a function. Esp that this function is only called in one place.